frodare/addon-redux
# Reset Redux state when switching stories

This makes addon-redux reset the Redux state to its initial state when switching stories (fixes #14 and #26 )

# Test PR

1. `npm run start`
2. Go to `SharedState/Writer` story
3. Click the button to increment the state stored in the Redux store
4. Go to `SharedState/Reader` story and notice the state if properly reset to `0`

# Reproduce previous bug

If you want to see the bug this PR fixes:

1. Comment out `[STORY_CHANGED]: (action: AnyAction) => store.dispatch({ type: ACTIONS_TYPES.RESET_REDUX_TYPE }),`
2. Follow the steps in the "Test PR" section

Notice that the `SharedState/Reader` state carried over state from `SharedState/Writer` which is why we need to reset state like this PR does
